### ISSUE TITLE
Failed to extract the starterkit

### ISSUE THREAD

**Issue by LadIQe** [fourkitchens/emulsify#307]
Hi guys, Im new in gulp and emulsify, using windows 10 1703, php 7.1 and nvm for windows.

In our project, we need to use emulsify, but I have problem to install it. Everything looks ok when emulsify is installing, but when pattern-lab is installing, there is problem with extracting starterkit:

```
suggested starterkits that work with this edition:

  1: pattern-lab/starterkit-twig-base
  2: pattern-lab/starterkit-twig-demo


downloading the starterkit. this may take a few minutes...
finished downloading the starterkit...
failed to extract the starterkit. easiest solution is to manually download it and copy <path>./dist</path to ./source/...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! emulsify@1.0.0 postinstall: `rm -rf pattern-lab && composer create-project -n pattern-lab/edition-twig-standard pattern-lab && rm -rf pattern-lab/source && ln -s ../components pattern-lab/source && git clone https://github.com/drupal-pattern-lab/bem-twig-extension.git && cp bem-twig-extension/bem.function.php components/_twig-components/functions && rm -rf bem-twig-extension/`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the emulsify@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above. 
```

I tried to download and copy dist from git, but when I type npm start, there is error for missing 'clean' task in gulpfile.js

Thanks for suggestions

**Comment by ModulesUnraveled** [comment]
There's a PR for better Windows support. Check that out and see if it helps : https://github.com/fourkitchens/emulsify/pull/300

**Comment by LadIQe** [comment]
ok so i tried your suggestion and again it doesnt work:

```
suggested starterkits that work with this edition:

  1: pattern-lab/starterkit-twig-base
  2: pattern-lab/starterkit-twig-demo


downloading the starterkit. this may take a few minutes...
finished downloading the starterkit...
failed to extract the starterkit. easiest solution is to manually download it and copy <path>./dist</path to ./source/...
🚫 Vendor directory not found.
💻 Please run `composer install` and run this again.
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN emulsify@1.0.0 No repository field.
npm WARN emulsify@1.0.0 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 5 packages from 4 contributors, updated 3 packages and audited 20375 packages in 68.877s
found 20 vulnerabilities (4 low, 7 moderate, 8 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by bisslad** [comment]
This seems to be an ongoing issue, particularly on windows. 

It seems to be related to a broken path being built within pattern-lab on windows. You end up with a path in your patternlab config which is half unix style, half windows style.

I found last night that by being super quick in opening the config file as soon as it is generated and fixing the path for: 

styleguideKitPath: 'C:\wamp\www\drupal8\web\themes\custom\bline\pattern-lab\vendor/pattern-lab/styleguidekit-twig-default'

to:

styleguideKitPath: 'vendor/pattern-lab/styleguidekit-twig-default'

I.e. a relative path, that you can get it to install.

I've been banging my head up against a wall for a couple of weeks now trying to get a clean install process on windows so hopefully that can save you at least a bit of trouble until the fix is merged in.

**Comment by LadIQe** [comment]
thank you guys for you suggestions.

but I moved to Ubuntu 18.10 and all my problems are now gone ;) everything works now

**Comment by evanmwillhite** [comment]
Recently ran into this issue as well. tl;dr is that we did the following:

1. Remove any existing/broken `pattern-lab` directory from your theme
1. Download a [pre-built Installation](https://github.com/pattern-lab/edition-php-twig-standard#download-a-pre-built-project) of Pattern Lab Standard Edition for Twig 
1. Untar/zip into a new `pattern-lab` directory
1. Remove the `pattern-lab/source` directory
1. Symlink your theme's `components` directory as the `pattern-lab/source` directory (on Windows we used `mklink /d pattern-lab\source ..\components` as an Administrator from the theme directory)

After that, we ran into a `Fatal error: Cannot declare class Symfony\Component\Yaml\Yaml` issue which required us to run `composer update`

Once we had done those steps, `npm start` worked properly. 

Background is that I believe there may be an issue with `composer create-project` running [these installation steps](https://github.com/pattern-lab/edition-php-twig-standard/blob/develop/composer.json#L33-L52) in the Pattern Lab Standard Edition, but I don't know which one. The error message didn't appear to provide that information.
